commands/configure/index.js

```diff
@@ -108,6 +106,15 @@ program
       const downloaded = await experienceDownload(null, {}, loadedConfig);
       if (downloaded) {
         logResult('success', 'Downloaded all experience resources!', 'green');
+      } else {
+        const { shouldBootstrap } = await inquirer.prompt([{
```

Comment:
before you prompt, check the FTUE. if they have already done or skipped bootstrapping, don't ask.

Reply:
also, if they say no, you should then set the FTUE flag on the application to skipped, just like the frontend